x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/moving_average.tsx

```diff
@@ -191,15 +191,15 @@ const MovingAveragePopup = () => {
       <p>
         <FormattedMessage
           id="xpack.lens.indexPattern.movingAverage.basicExplanation"
-          defaultMessage="The Moving Average slides a window across the data and emits the average value of that window."
+          defaultMessage="Moving average slides a window across the data and displays the average value in the window."
         />
       </p>
 
       <p>
         <FormattedMessage
           id="xpack.lens.indexPattern.movingAverage.longerExplanation"
-          defaultMessage="The Lens Moving Average uses a simple arithmetic mean of the window and applies a skip policy for gaps: 
-            this means that for missing values the bucket is skipped and the calculation is performed on the next one."
+          defaultMessage="To calculate the moving average, **Lens** uses the mean of the window and applies a skip policy for gaps.
```

Comment:
Markdown isn't supported here, so the bold doesn't show. There's a few places we need to update this.